tests/components/Microsoft.DotNet.UpgradeAssistant.Tests/Microsoft.DotNet.UpgradeAssistant.Tests.csproj

```diff
@@ -3,6 +3,7 @@
     <TargetFramework>net5.0</TargetFramework>
   </PropertyGroup>
   <ItemGroup>
+    <ProjectReference Include="..\..\..\src\components\Microsoft.DotNet.UpgradeAssistant.Analysis\Microsoft.DotNet.UpgradeAssistant.Analysis.csproj" />
```

Comment:
I'm guess this is not needed here anymore?

Reply:
removed :)